mkosi/__init__.py

```diff
@@ -2836,6 +2836,10 @@ def install_debian_or_ubuntu(args: MkosiArgs, root: Path, *, do_run_build_script
             run_workspace_command(args, root, ["kernel-install", "add", kver, Path("/") / kimg])
 
     root.joinpath("etc/locale.conf").write_text("LANG=C.UTF-8\n")
+    try:
```

Comment:
wouldn't it be better to check if it exists, and avoid overwriting it? So that the distro configuration is what gets applied to the image?

Reply:
At least on Ubuntu Focal, /etc/default/locale is populated as follows:

```
root@image:~# cat /etc/default/locale 
#  File generated by update-locale
```

which doesn't seem exactly useful

Reply:
Isn't `/etc/default/locale` rewritten all the time when one does `dpkg-reconfigure locales`? Does debconf have any problems when the file is a symlink? 

It's been a while I last checked the patches for localectl in Debian, but I vaguely remember those using `/etc/default/locale` and `/etc/default/keyboard` in addition to what localectl uses by upstream. 

If the symlink doesn't pose a problem, I think it would be preferable to have a single file instead of two and to keep Debianisms to what's absolutely necessary.